mods/cnc/weapons/explosions.yaml

```diff
@@ -1,7 +1,28 @@
+^DamagingExplosion:
+	Warhead@1Dam: SpreadDamage
+		Spread: 426
+		Damage: 40
+		Versus:
+			None: 90
+			Wood: 75
+			Light: 60
+			Heavy: 25
+		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
+	Warhead@2Eff: CreateEffect
+		Explosions: poof
+		ImpactSounds: xplos.aud
+		TargetSearchRadius: 0
+	Warhead@3Smu: LeaveSmudge
+		SmudgeType: Crater
+		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
+
 FlametankExplode:
+	Inherits: ^DamagingExplosion
 	Warhead@1Dam: SpreadDamage
 		Spread: 1c0
 		Damage: 115
+		Versus:
```

Comment:
I can test it once this gets merged. The vs heavy was useful for APCs and didn't do a lot of damage as it was vs tanks. But the vs none to 90 and light vs 60 is confusing since they are doing less then 100 damage which was the base damage before its buff in last release.